tests/src/test/scala/org/http4s/UriSpec.scala

```diff
@@ -233,10 +220,10 @@ http://example.org/a file
       Uri(Some(scheme"mailto"), path = "John.Doe@example.com").toString must_== ("mailto:John.Doe@example.com")
     }
 
-    "render an URL with username and password" in {
+    "render a URL with username and password" in {
       Uri(
         Some(Scheme.http),
-        Some(Authority(Some(userInfo"username:password"), host"some.example.com", None)),
+        Some(authority"username:password@some.example.com"),
```

Comment:
There are more of these, but I got bored porting them.  I think most could use the URI macro, which is what most people will use, and what I really want to test hard.